src/test/java/org/sert2521/gamename/Drivetrain/DriveStraightTest.kt

```diff
@@ -0,0 +1,84 @@
+package org.sert2521.gamename.Drivetrain
+
+import org.fest.assertions.Assertions.assertThat
+import org.fest.assertions.Delta
+import org.junit.Before
+import org.junit.Test
+import org.sert2521.gamename.Drivetrain.commands.DriveStraight
+import org.strongback.command.CommandTester
+import org.strongback.components.Motor
+import org.strongback.components.Motor.compose
+import org.strongback.drive.TankDrive
+import org.strongback.mock.Mock
+
+/**
+ * Created by AMD on 6/17/17.
+ */
+class DriveStraightTest() {
+    private val TOLERANCE = Delta.delta(0.001)
+    private val START_TIME_MS: Long = 1000
```

Comment:
This recreates those properties every time the class is created so they are not constants. Define them like this at the bottom of the class:
```kotlin
private companion object {
    val TOLERANCE = Delta.delta(0.001)
    val START_TIME_MS = 1000L
}
```